docs-ref-mapping/reference.yml

```diff
@@ -273,6 +273,7 @@
       - 'azure-mgmt-datalake-store*'
     - name: Client
       uid: azure.python.sdk.landingPage.services.Data.Lake.Store.Client
+      href: ~/docs-ref-services/adls-gen1-client.md
```

Comment:
@akharit I think you need to delete the landingPageType entry below this new href line. This will allow the TOC to render the new page properly under Client. Please let me know when you've applied the fix and I'll check again.